library/src/main/java/com/novoda/downloadmanager/lib/Query.java

```diff
@@ -228,6 +219,14 @@ private String joinStrings(String joiner, Iterable<String> parts) {
         return builder.toString();
     }
 
+    private static String[] longArrayToStringArray(long[] longs) {
```

Comment:
moving from `DownloadManager.longArrayToStringArray` makes sense :+1:  can the old one be deleted? :dart: